src/migration/migration.cpp

```diff
@@ -100,7 +100,7 @@ int Migration::GetCurrentVersion()
 void Migration::MergeMigrationFiles(const std::string& migrationDir)
 {
     if (!std::filesystem::exists(migrationDir)) {
-        AOS_ERROR_THROW("migration path doesn't exist (" + migrationDir + ")", aos::ErrorEnum::eInvalidArgument);
+        AOS_ERROR_THROW(aos::ErrorEnum::eInvalidArgument, "migration path doesn't exist (" + migrationDir + ")");
```

Comment:
redundant namespace usage
```suggestion
        AOS_ERROR_THROW(ErrorEnum::eInvalidArgument, "migration path doesn't exist (" + migrationDir + ")");
```

Reply:
Done